types/leaflet/index.d.ts

```diff
@@ -241,20 +241,20 @@ export abstract class Evented extends Class {
      */
     on(eventMap: LeafletEventHandlerFnMap): this;
 
-    /* tslint:disable:unified-signatures */ // With an eventMap there are no additional arguments allowed
     /**
      * Removes a previously added listener function. If no function is specified,
      * it will remove all the listeners of that particular event from the object.
      * Note that if you passed a custom context to on, you must pass the same context
      * to off in order to remove the listener.
      */
+    // tslint:disable-next-line
```

Comment:
Why do you put all these `disable-next-line` comments to the definition? Please remove them!

Reply:
Ah, ok I will test it and give a reply...

Reply:
Disabling is probably necessary here if we want to make a good distinction between these two overloads. I suggest replacing `// tslint:disable-next-line` with `// tslint:disable-next-line:unified-signatures` to be explicit about what we're disabling and keeping the `// With an eventMap there are no additional arguments allowed` comment to explain why we're doing it.